extensions/standard-processors/tests/integration/TLSServerSocketSupportedProtocolsTest.cpp

```diff
@@ -194,21 +195,21 @@ class SimpleSSLTestClient  {
 class SimpleSSLTestClientTLSv1  : public SimpleSSLTestClient {
  public:
   SimpleSSLTestClientTLSv1(const std::string& host, const std::string& port)
-      : SimpleSSLTestClient(TLSv1_client_method(), host, port) {
+      : SimpleSSLTestClient(SSL_OP_NO_SSLv3 | SSL_OP_NO_TLSv1_2 | SSL_OP_NO_TLSv1_1 | SSL_OP_NO_TLSv1_3, host, port) {
```

Comment:
[this](https://www.openssl.org/docs/man3.1/man3/SSL_CTX_set_options.html) documentation claims that these are deprecated and that `SSL_CTX_set_min_proto_version` and `SSL_CTX_set_max_proto_version` should be used instead

Reply:
Updated to set_max and set_min in 67eb58112fe948432498c39cae18aec78251fd22